app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class LikesController < ApplicationController
+  before_action :current_user, only: [:create]
+
+  def create
+    @post = Post.find(params[:post_id])
+    new_like = current_user.likes.new(author_id: current_user.id, post_id: @post.id)
+
+    redirect_to "/users/#{@post.author_id}/posts/#{@post.id}", notice: 'Like created' if new_like.save
+  end
```

Comment:
- [ ] [OPTIONAL] You did great adding the like functionality but I think to improve it would be better to like a post only once because for now you can like a post multiple times